### DISCUSSION TITLE
NvChad + Discord Presence

### DISCUSSION THREAD

**Discussion by karastoyanov** [NvChad/NvChad#1763] [Q&A]
I am trying to install and run [andweeb/presence.nvim](https://github.com/andweeb/presence.nvim) plugin. I managed to activate the plugin in init.lua, but every time i close and open again the NvChad I need to run :PackerSync to activate the plugin before it starts displaying anything inside Discord. How can I make the plugin to load by default when I open nvim?

OS: Linux Debian
Neovim 0.9.0

**Comment by siduck** [comment]
show what you did

**Comment by karastoyanov** [comment]
I just copied `['andweeb/presence.nvim'] = {},` inside `~/.config/nvim/lua/plugis/init.lua`. Everything seems to work, until I close nvim and re-open it, the plugin doesn't load automatically and I need to re-sync again.

**Reply by siduck** [reply]
@karastoyanov thats not the right away to install a custom plugin in nvchad, please read the docs. And dont expect neovim to hot reload config, you need to reopen neovim to see the changes

**Comment by 2giosangmitom** [comment]
do you have this in your config
require("presence").setup()
